Add mintso config parameters (#19452)
pingcap/docs: tiflash/tiflash-configuration.md

    ## New in v7.4.0. This item controls whether to enable the TiFlash resource control feature. When it is set to true, TiFlash uses the pipeline execution model.
    enable_resource_control = true

>     ## New in v6.0.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of threads that one resource group could use. The default value is 5000.
>     task_scheduler_thread_soft_limit = 5000
> 
>     ## New in v6.0.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of threads in global scope. The default value is 10000.
>     task_scheduler_thread_hard_limit = 10000
> 
>     ## New in v6.4.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of running queries in a TiFlash instance. The default value is 0 which means two times the vcpu.
>     task_scheduler_active_set_soft_limit = 0

Reviewer: ```suggestion
    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs.
    task_scheduler_active_set_soft_limit = 0
```

Author: > ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_soft_limit = 5000
>     ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000.
>     task_scheduler_thread_hard_limit = 10000
>     ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs.
>     task_scheduler_active_set_soft_limit = 0

done

>     ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_soft_limit = 5000
> 
>     ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_hard_limit = 10000
> 
>     ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_active_set_soft_limit = 0

Reviewer: ```suggestion
    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_active_set_soft_limit = 0
```

>     ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_soft_limit = 5000
> 
>     ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_hard_limit = 10000
> 
>     ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_active_set_soft_limit = 0

Reviewer: ```suggestion
    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_active_set_soft_limit = 0
```
